portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/searchtuning/SearchExperiences.testcase

```diff
@@ -0,0 +1,601 @@
+				"decay": 0.3
+			}
+		},
+		"boost": 100
+	}
+}
+			''';
+
+		SearchExperiences.addQueryElements(
+			elasticsearchQuery = "${elasticsearchQueryJSON}",
+			fragment = "Paste any Elasticsearch query");
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.addWidgets(searchPortletList = "Blueprints Options");
+
+		SearchExperiences.selectBlueprintFromOptionsWidget(name = "The Closer to Liferay, the Better");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "Custom Fields");
+
+		CustomFields.addCP(
+			customFieldName = "location",
+			customFieldType = "Geolocation",
+			resourceName = "Web Content Article");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Web Content");
+
+		WebContentNavigator.gotoAddCP();
+
+		Click(locator1 = "TextInput#TITLE");
+
+		PortletEntry.inputTitle(title = "Closest Location");
+
+		PortletEntry.publish();
+
+		WebContentNavigator.gotoAddCP();
+
+		Click(locator1 = "TextInput#TITLE");
+
+		PortletEntry.inputTitle(title = "Closer Location");
+
+		Panel.expandPanel(panel = "Categorization");
+
+		DragAndDrop.javaScriptDragAndDropToObject(
+			locator1 = "WCEditWebContent#GEOLOCATION_LEAFLET_MARKER_DRAG_HANDLE",
+			locator2 = "WCEditWebContent#GEOLOCATION_ZOOM_OUT");
+
+		PortletEntry.publish();
+
+		WebContentNavigator.gotoAddCP();
+
+		Click(locator1 = "TextInput#TITLE");
+
+		PortletEntry.inputTitle(title = "Close Location");
+
+		Panel.expandPanel(panel = "Categorization");
+
+		Click(locator1 = "WCEditWebContent#GEOLOCATION_ZOOM_OUT");
+
+		DragAndDrop.javaScriptDragAndDropToObject(
+			locator1 = "WCEditWebContent#GEOLOCATION_LEAFLET_MARKER_DRAG_HANDLE",
+			locator2 = "WCEditWebContent#GEOLOCATION_ZOOM_OUT");
+
+		PortletEntry.publish();
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "location");
+
+		SearchResultPortlet.viewSearchResultsSpecificOrder(resultsList = "Closest Location,Closer Location,Close Location");
+	}
+
+	test CreateBoostAssetType {
+		property test.name.skip.portal.instance = "SearchExperiences#CreateBoostAssetType";
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(name = "My Blueprint");
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.addWidgets(searchPortletList = "Blueprints Options");
+
+		SearchExperiences.selectBlueprintFromOptionsWidget(name = "My Blueprint");
+
+		JSONWebcontent.addWebContent(
+			content = "Liferay",
+			groupName = "Guest",
+			title = "Liferay Web Content");
+
+		JSONBlog.addEntry(
+			entryContent = "Liferay Liferay",
+			entryTitle = "Liferay Blogs Entry");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
```

Comment:
Since the test should still be open at the search page, no, but I think a refresh might be required since we usually need to refresh after configuring search widgets